drivers/virtualbox/virtualbox_test.go

```diff
@@ -63,6 +63,27 @@ func TestDefaultSSHUsername(t *testing.T) {
 	assert.Equal(t, "docker", username)
 }
 
+func TestParseShareFolder(t *testing.T) {
+	shareDir, shareName := parseShareFolder("dir:name")
```

Comment:
Can you refactor this into one `TestParseShareFolder` function with a [test table](https://github.com/golang/go/wiki/TableDrivenTests)?

e.g., you'd do something like:

```go
testCases := []struct{
    input, expectedShareDir, expectedShareName string
}{
   {"dir:name", "dir", "name"}
   // ...
}
```